dango/types/src/lending.rs

```diff
@@ -0,0 +1,65 @@
+use {
+    grug::{Addr, Coins, Denom, Part},
+    std::{collections::BTreeMap, sync::LazyLock},
+};
+
+/// The namespace that tokens associated with lending will be minted under.
+/// The lending contract must be granted admin power over this namespace.
+pub static NAMESPACE: LazyLock<Part> = LazyLock::new(|| Part::new_unchecked("lending"));
+
+/// Sub-namespace that liquidity share tokens will be minted under.
+pub static SUBNAMESPACE: LazyLock<Part> = LazyLock::new(|| Part::new_unchecked("pool"));
```

Comment:
Defining these as global static, so that we don't need to `Part::from_str` every time we need to use them.

This is not consistent in other the contracts that deal with namespaces (amm, ibc-transfer, token-factory). Will address this in a follow up PR.